openqaoa/optimizers/result.py

```diff
@@ -34,6 +34,38 @@ def most_probable_bitstring(cost_hamiltonian, measurement_outcomes):
     return {'solutions_bitstrings': solutions_bitstrings,
             'bitstring_energy': bitstring_energy(cost_hamiltonian, solutions_bitstrings[0])}
 
+def lowest_cost_bitstrings(cost_hamiltonian:Hamiltonian, measurement_outcomes:dict, n_bitstrings:int = 1) -> dict:
```

Comment:
perhaps it makes sense to add this function as a method of the class `Result`. Also, this way the user can call the method by passing the argument `n_bitstrings` when needed and can save computation (performed inside the __init__) when not required.

Reply:
I agree. I'll do so. Do you think I should make the same for the other function `most_probable_bitstring`?

Reply:
I think, probably @Q-lds remembers the reason why we decided to put this function outside the `Result` class. I believe there was a reason but I can't remember.

Reply:
I think we were using it in the tests as a in individual function (otherwise, we may turn it into a class module?)

Reply:
Right okay! I think let's leave that function in place for now for minimal changes through this PR.